api/models/orderDao.js

```diff
@@ -0,0 +1,49 @@
+const { AppDataSource } = require('./data_source');
+
+const addOrder = async (userId) => {
+    await AppDataSource.query(
```

Comment:
```suggestion
    await appDataSource.query(
```
instance naming에는 대문자로 시작하지 않도록 해주세요! 일반적으로 자바스크립트에서는 특별한 경우를 제외하고는 class 만이 대문자로 시작하는것이 컨벤션입니다~